.github/workflows/auto-updates.yaml

```diff
@@ -197,7 +195,7 @@ jobs:
         # If we don't run this before the "git diff-index" it seems to list
         # every file that's been touched by codegen.
         git status
-        echo "create_pr=${{ needs.meta.outputs.pr_empty_deps }}" >> $GITHUB_ENV
+        echo "create_pr=${{ github.events.inputs.pr-empty-deps }}" >> $GITHUB_ENV
```

Comment:
double checking... `pr-empty-dep` or `pr_empty_dep` ?

Reply:
ah nevermind, it was renamed. 👍

Reply:
GitHub seemed to like `-` better than `_`. /shrug